CMakeLists.txt

```diff
@@ -53,7 +53,10 @@ target_include_directories(secp256k1 PRIVATE secp256k1 INTERFACE secp256k1/inclu
 
 # Compiler options
 if(MSVC)
-  add_compile_options(/W4 /WX)
```

Comment:
Shall we keep /WX?

Reply:
Yes we can but have to resolve several warnings

Reply:
I see